pom.xml

```diff
@@ -125,6 +126,11 @@
           <version>${assertj.version}</version>
           <scope>test</scope>
       </dependency>
+      <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+          <version>${guava.version}</version>
```

Comment:
If you are not using plugin-first class loading or classpath masking, this is wrong. [Details](https://jenkins.io/doc/developer/plugin-development/dependencies-and-class-loading/)

Reply:
#322